package.nls.json

```diff
@@ -390,6 +390,7 @@
     "aws.cloudWatchLogs.codeLens.loadOlder": "Load older events...",
     "aws.cloudWatchLogs.codeLens.loadNewer": "Load newer events...",
     "aws.cloudWatchLogs.invalidEditor": "Not a Cloudwatch Log stream: {0}",
+    "aws.cloudWatchLogs.limit.desc": "The amount of log entries pulled per request from CloudWatch Logs (max 10000)",
```

Comment:
```suggestion
    "aws.cloudWatchLogs.limit.desc": "Number of log entries pulled per request from CloudWatch Logs (max 10000)",
```

Reply:
Just as a heads-up, "The amount of..." is terminology we have for other similar settings. May be worth doing a full pass on these at some point.

Reply:
Also if you want to be technically correct you can get less entries even with > 10000 entries since there's also a size limit, the new wording makes more sense though